src/main/java/module-info.java

```diff
@@ -65,4 +65,5 @@
     requires de.saxsys.mvvmfx.validation;
     requires richtextfx;
     requires unirest.java;
+    requires io.github.classgraph;
```

Comment:
Move to https://github.com/JabRef/jabref/blob/master/src/test/java/module-info.test